### PR TITLE
Cn0511 rebased

### PR DESCRIPTION
**Pull request by VBeleca**
# Description

- Added calibration of the output amplitude from 0 to 5.5GHz and exposed the raw attribute of the trim DAC (ad5693r) in adi/cn0511.py. 
 - Added an example of enabling and disabling the output as well as setting the calibrated output amplitude and frequency in adi.cn0511_example.py.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How has this been tested?

To test these changes, examples/cn0511_example.py was run locally on a calibrated rpi+cn0511. The output of cn0511 was measured for different combinations of output powers and frequencies with Keysight spectrum analyzer N9030B.
 - test/test_cn0511_p.py needs to be updated.

**Test Configuration**:
* Hardware: cn0511+rpi
* OS: Kuiper Linux

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by VBeleca** [comment]
Force push: a comment was modified in examples/cn0511_example.py and a typo was corrected in adi/cn0511.py at  "trim_frequency_raw" attribute.

**Comment by tfcollins** [comment]
@VBeleca the main issue here is that we need libad9166+bindings installed in the CI system to even import the adi.ad9166 class. However, that requirement should be optional (similar to libad9361) since not everyone should need to install that library to use pyadi-iio itself.

After this https://github.com/analogdevicesinc/pyadi-iio/pull/351 is merged you can rebase to get the library on the linux side. However, since libad9166 doesn't have Windows builds you will still need to make importing libad9166 optional. Take a look here on an example with libad9361: 
https://github.com/analogdevicesinc/pyadi-iio/blob/master/adi/fmcomms5.py#L38
https://github.com/analogdevicesinc/pyadi-iio/blob/master/adi/fmcomms5.py#L80

File: adi/cn0511.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+        # #Print offset calibration constants:
+        # print("\n\nOffsets calibration constants: ")
+        # for offset in self.__calibration_data.contents.Offsets[:self.__calibration_data.contents.Len]:
+        #     print(offset, end=", ")
+
+        # #Print gain calibration constants:
+        # print("\n\nGains calibration constants: ")
+        # for gain in self.__calibration_data.contents.Gains[:self.__calibration_data.contents.Len]:
+        #     print(gain, end=", ")
```
Comment (tfcollins) [inline adi/cn0511.py:73]: Please remove these comments

**Review by tfcollins — CHANGES_REQUESTED** [review]
Please be sure to run the precommit checks: https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#precommit

File: adi/cn0511.py (hunk 1 above)
Comment (VBeleca) [reply adi/cn0511.py]: done

File: adi/ad916x.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+# INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT, MERCHANTABILITY AND FITNESS FOR A
+# PARTICULAR PURPOSE ARE DISCLAIMED.
+#
+# IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, INTELLECTUAL PROPERTY
+# RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+# BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+# THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+import iio
```
[the remark this answers is not in the document]
Comment (VBeleca) [reply adi/ad916x.py]: done

**Pull request merged**